server/game/cards/08-GT/FranciscoRosales.js

```diff
@@ -0,0 +1,40 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+const StandardActions = require('../../PlayActions/StandardActions.js');
+
+class FranciscoRosales extends DudeCard {
+    setupCardAbilities(ability) {
+        this.job({
+            title: 'Francisco Rosales',
+            playType: 'noon',
+            cost: ability.costs.bootLeader(),
+            target: 'townsquare',
+            message: context =>
+                this.game.addMessage('{0} uses {1} to mark town square for a job', context.player, this),
+            onSuccess: (job, context) => {
+                this.game.resolveGameAction(
+                    GameActions.search({
+                        title: 'Select a horse or sidekick for Francisco',
```

Comment:
No need to mention Francisco, his name will be in the title of the prompt.
```suggestion
                        title: 'Select a horse or sidekick',
```